src/jit/compiler.h

```diff
@@ -5606,6 +5606,10 @@ protected :
                             O2K_SUBRANGE };
     struct AssertionDsc
     {
+        AssertionDsc()
+        {
+          memset(this, 0, sizeof(AssertionDsc));
```

Comment:
The memset for that type was already present and I just moved it to the constructor.
I think clang will do alright with that.

I'm just fixing an issue which appeared after my commit, though not related to it ;)